src/components/ResultsItemDetails.vue

```diff
@@ -49,12 +46,19 @@ export default {
   methods: {
     savePark() {
       this.$store.commit('savePark', this.park)
+    },
+    determinePhoto() {
```

Comment:
Thanks for your hard work on this. I know getting this sorted out was a tough challenge thanks to the way google's API works.